hatch/src/cli/commands/new.rs

```diff
@@ -63,6 +78,13 @@ impl<'command> Command<'command> for New {
            .help("Set the project version")
            .long("version").short("v").takes_value(true)
            .required(false))
+
+      .arg(Arg::with_name("include")
+        .help("List GitHub dependencies in format username/project[:branch]")
```

Comment:
Can we indent these such that they align with the the others?

Reply:
There are five spaces in front of each of the others. I can make my addition consistent, but that still bugs me a little. Everything else is indented two spaces.

Reply:
This help comment also should be github agnostics.

Reply:
So they are indented like they are because those properties belong to the Arg not the .arg method.